app/models/game.rb

```diff
@@ -6,6 +6,7 @@ class Game < ActiveRecord::Base
   belongs_to :sith_user, foreign_key: 'sith_user_id', class_name: "User"
   has_many :ships
 
+  scope :game, -> { where("jedi_user_id is NULL or sith_user_id is NULL")}
```

Comment:
Another really small nitpick is the name of the scope being `game` is a bit unusual.  I might call it something like:

`unjoined_games` to adequately describe what is being scoped.  Besides that it's perfect.